renku/core/commands/providers/renku.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 - Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Renku dataset provider."""
+import attr
+
+from renku.core import errors
+from renku.core.commands.providers.api import ExporterApi, ProviderApi
+from renku.core.models.git import GitURL
+
+
+@attr.s
+class RenkuProvider(ProviderApi):
+    """Renku API provider."""
+
+    is_doi = attr.ib(default=False)
+    _accept = attr.ib(default='application/json')
+
+    @staticmethod
+    def supports(uri):
+        """Whether or not this provider supports a given uri."""
+        return 'renku' in uri.lower()
+
+    def find_record(self, uri, client):
```

Comment:
```suggestion
    def find_record(self, uri, client=None):
```

My linter complains that the signature is not the same. :smile_cat:

Reply:
Or instead update the `ProviderApi` interface